src/styles/_variables.scss

```diff
@@ -0,0 +1,24 @@
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@700&display=swap');
+$darkBlue: #013578;
+$lightBlue: #33a6f2;
+
+
+@mixin rowDisplay {
```

Comment:
Awesome to see we are already starting to make some mixins. Great thinking and code looks great!